go.mod

```diff
@@ -188,6 +188,7 @@ require (
 )
 
 replace (
+	github.com/google/cel-go => github.com/kcp-dev/cel-go v0.0.0-20230104211646-409eafa0bc12
```

Comment:
Couldn't we just set the version to be the commit here, or do we need the fork?

Reply:
It hasn't been backported yet upstream

Reply:
We need to be on 0.12 though